docs/readthedocs/source/doc/LLM/Quickstart/open_webui_with_ollama_quickstart.md

```diff
@@ -0,0 +1,125 @@
+
+<video src="https://llm-assets.readthedocs.io/en/latest/_images/open_webui_demo.mp4" width="100%" controls></video>
+
+## Quickstart
+This quickstart guide walks you through setting up and using the [open-webui](https://github.com/open-webui/open-webui) with Ollama. 
+
+
+### 1 Run Ollama on Linux with Intel GPU
+
+To use the open-webui on Intel GPU, first ensure that you can run Ollama on Intel GPU. Follow the instructions on the [Run Ollama on Linux with Intel GPU](ollama_quickstart.md). Please keep ollama service running after completing the above steps.
+
+### 2 Install and Run Open-Webui
+
+#### Option 1: Docker
+
+> [!IMPORTANT]
+> Include `-v open-webui:/app/backend/data` in your Docker command when using Docker for Open WebUI. This ensures the database is properly mounted and prevents data loss.
+
+- **For Ollama on Your Computer**:
+
+  ```bash
+  export no_proxy=localhost,127.0.0.1
+  docker run -d -p 3000:8080 --add-host=host.docker.internal:host-gateway -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
+  ```
+
+- **For Ollama on a Different Server**:
+
+  Replace `OLLAMA_BASE_URL` with your server's URL:
+
+  ```bash
+  export no_proxy=localhost,127.0.0.1
+  docker run -d -p 3000:8080 -e OLLAMA_BASE_URL=https://example.com -v open-webui:/app/backend/data --name open-webui --restart always ghcr.io/open-webui/open-webui:main
+  ```
+
+- Access open-webui at [http://localhost:3000](http://localhost:3000). 
+
+#### Option 2: Manual Build
+
+**Requirements** 
+
+- Node.js (>= 20.10) or Bun (>= 1.0.21)
+- Python (>= 3.11)
+
+**Installation Steps**
+
+```sh
+# Optional: Use Hugging Face mirror for restricted areas
+export HF_ENDPOINT=https://hf-mirror.com
+
+export no_proxy=localhost,127.0.0.1
+
+git clone https://github.com/open-webui/open-webui.git
+cd open-webui/
+cp -RPp .env.example .env  # Copy required .env file
+
+# Build Frontend
+npm i
+npm run build
+
+# Serve Frontend with Backend
+cd ./backend
+pip install -r requirements.txt -U
+bash start.sh
+```
+
+open-webui will be accessible at http://localhost:8080/.
+
+> For detailed information, visit the [open-webui official repository](https://github.com/open-webui/open-webui).
+>
+
+### 3. Using Open-Webui
+
+#### Log-in and Pull Model
+
+If this is your first time using it, you need to register. After registering, log in with the registered account to access the interface.
+
+<a href="https://llm-assets.readthedocs.io/en/latest/_images/open_webui_signup.png" target="_blank">
+  <img src="https://llm-assets.readthedocs.io/en/latest/_images/open_webui_signup.png" width="100%" />
+</a>
+
+
+<a href="https://llm-assets.readthedocs.io/en/latest/_images/open_webui_login.png" target="_blank">
+  <img src="https://llm-assets.readthedocs.io/en/latest/_images/open_webui_login.png" width="100%" />
+</a>
+
+Check your ollama service connection in `Settings`. If everything goes well, you will get a message as shown below.
```

Comment:
- add steps to set ollama url (if not on the same computer)
- there's only a pic illustrating the "after setting" - add a pic illustrating "before setting"